src/TrafficSign/TrafficSign.types.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Properties of the TrafficSign component
+ */
+
+export type TrafficSignProps = {
+  /**
+   * The type of the traffic sign
+   */
+  type:
+    | "barrier"
+    | "barrier2S"
+    | "beaconArrowL"
+    | "beaconArrowR"
+    | "beaconHatchedWide"
+    | "beaconHatchedWideL"
+    | "beaconHatchedWideR"
+    | "beaconL"
+    | "beaconL2S"
+    | "beaconR2S"
+    | "beaconR"
+    | "caution"
+    | "children"
+    | "crossroads"
+    | "curveL"
+    | "curveR"
+    | "cyclistsCrossing"
+    | "cyclistsCrossingL"
+    | "cyclistsCrossingR"
+    | "deadEnd"
+    | "directionBoardL"
+    | "directionBoardR"
+    | "directionBoardWideL"
+    | "directionBoardWideR"
+    | "diversionL"
+    | "diversionR"
+    | "endOfLimitations"
+    | "exitHighway"
+    | "exitInnerCityRoad"
+    | "exitLimitedAccessRoad"
+    | "fileUnknown"
+    | "giveWay"
+    | "highway"
+    | "highwayEnd"
+    | "icy"
+    | "limitedAccessRoad"
+    | "limitedAccessRoadEnd"
+    | "livingStreet"
+    | "livingStreetEnd"
+    | "minimumSpeed"
+    | "minimumSpeedEnd"
+    | "narrowRoad"
+    | "narrowRoadL"
+    | "narrowRoadR"
+    | "noEntry"
+    | "noOvertaking"
+    | "noOvertakingEnd"
+    | "noOvertakingTrucks"
+    | "noOvertakingTrucksEnd"
+    | "noTraffic"
+    | "oneWayLeft"
+    | "oneWayRight"
+    | "oneWayStraight"
+    | "parkingGarage"
+    | "parkingLot"
+    | "passL"
+    | "passR"
+    | "pedestrianCrossing"
+    | "pedestrians"
+    | "priorityRoad"
+    | "priorityRoadEnd"
+    | "rightOfWay"
+    | "roadWorks"
+    | "roundabout"
+    | "sCurveL"
+    | "sCurveR"
+    | "slipperyRoad"
+    | "speedLimit"
+    | "speedLimitEnd"
+    | "speedLimitZone"
+    | "speedLimitZoneEnd"
+    | "steepAscent"
+    | "steepDescent"
+    | "stop"
+    | "straightOrLeft"
+    | "straightOrRight"
+    | "trafficLight"
+    | "trafficQueues"
+    | "tunnel"
+    | "turnAheadL"
+    | "turnAheadR"
+    | "turnL"
+    | "turnR"
+    | "unevenRoad"
+    | "urbanArea"
+    | "urbanAreaEnd"
+    | "warningPedestrianCrossing"
+    | "weightLimit"
+    | "widthLimit";
+  /**
+   * The coordinates of the sign in Fr0 [x,y]
+   */
+  points: number[];
+  /**
+   * The value of the sign (e.g., speed limit)
+   */
+  value?: string;
```

Comment:
The combination of type and value seems weird to me. Also means I can enter unsupported values. Why not just allow speedLimit100, speedLimit60 etc as types and do away with value? The list becomes longer of course but it's already very long and you would then get proper type support.

<img width="1420" alt="image" src="https://github.com/IPG-Automotive-UK/react-ui/assets/8976377/d25a8821-e866-4802-bf3c-5828a2315998">

Reply:
After discussion, the value field has been removed. All signs are accessible only via the type string. In the future if support for custom text is added, it will be done via a `label` prop.